pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -426,7 +430,59 @@ class Config11 {
             buildArgs : [
                     openj9 : '--ssh --disable-adopt-branch-safety -r git@github.ibm.com:runtimes/openj9-openjdk-jdk11 -b ibm_sdk'
             ],
-            test                : 'default',
+            test                : [
+                    nightly: [
+                        'sanity.functional',
+                        'sanity.openjdk',
+                        'sanity.perf',
+                        'sanity.jck',
+                        'sanity.system',
+                        'extended.functional',
+                        'extended.openjdk',
+                        'special.system'
+                    ],
+                    weekly : [     
+                        'sanity.functional',
+                        'sanity.openjdk',
+                        'sanity.perf',
+                        'sanity.jck',
+                        'sanity.system',
+                        'extended.functional',
+                        'extended.openjdk',
+                        'extended.perf',
+                        'extended.jck',
+                        'extended.system',
+                        'special.functional',
+                        'special.jck',
+                        'special.openjdk',
+                        'special.system',
+                        'dev.functional',
+                        'extended.functional.fips140_3_OpenJCEPlusFIPS',
+                        'sanity.jck.fips140_3_OpenJCEPlusFIPS',
+                        'extended.jck.fips140_3_OpenJCEPlusFIPS',
+                        'special.jck.fips140_3_OpenJCEPlusFIPS',
+                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS',
+                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS'
+                    ],
+                    release : [
+                        'sanity.functional',
+                        'sanity.openjdk',
+                        'sanity.perf',
+                        'sanity.jck',
+                        'sanity.system',
+                        'extended.functional',
+                        'extended.openjdk',
+                        'extended.perf',
+                        'extended.jck',
+                        'extended.system',
+                        'special.functional',
+                        'special.jck',
+                        'special.openjdk',
+                        'special.system',
+                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS',
+                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS'
```

Comment:
We do not run release testing using the Semeru pipeline anymore. No need to add FIPS in release.